src/domain/Profile.ts

```diff
@@ -11,14 +11,14 @@ export const STATE = {
   DISCONNECTED: 'disconnected',
   AWAY: 'away',
 };
-export const LINE_STATE = {
-  AVAILABLE: 'available',
-  HOLDING: 'holding',
-  RINGING: 'ringing',
-  TALKING: 'talking',
-  UNAVAILABLE: 'unavailable',
-  PROGRESSING: 'progressing',
-};
+export enum LineState {
+  AVAILABLE = 'available',
+  HOLDING = 'holding',
+  RINGING = 'ringing',
+  TALKING = 'talking',
+  UNAVAILABLE = 'unavailable',
+  PROGRESSING = 'progressing',
+}
```

Comment:
```suggestion
export const LINE_STATE= {
  AVAILABLE: 'available',
  HOLDING: 'holding',
  RINGING: 'ringing',
  TALKING: 'talking',
  UNAVAILABLE: 'unavailable',
  PROGRESSING: 'progressing',
} as const;
```

enums have some issues in typescript, can we use this instead ?